Snakefile

```diff
@@ -0,0 +1,16 @@
+import os
+import glob
+
+# obtaining plate_ids
+sql_paths = glob.glob("./data/*.sqlite")
+PLATE_IDS = [Path(sql_file).stem for sql_file in sql_paths]
+
+include: "rules/preprocessing.smk"
+
+rule all:
+    input:
+        # expected outputs from the first DAG "Preprocessing"
+        expand("results/preprocessing/{plate_id}.aggregate.csv.gz", plate_id=PLATE_IDS),
```

Comment:
```suggestion
        expand("results/preprocessing/{plate_id}_aggregate.csv.gz", plate_id=PLATE_IDS),
```

For consistency

(Note, you can commit this suggestion directly without needing to implement yourself) Click `commit` below this comment!